src/services/encryption.ts

```diff
@@ -7,8 +6,8 @@ export function encryptPrivateKey(privateKey: string, password: string) {
 
 export function decryptPrivateKey(privateKey: string, password: string): string | never {
   const decryptedKey = AES.decrypt(privateKey, password).toString(enc.Utf8)
-  if (decryptedKey === '') {
-    throw new InputError('Wallet password is incorrect', 'wallet-unlock-password')
-  }
+  // if (decryptedKey === '') {
```

Comment:
this is being handled in the call location now from what I can see, so should we take out the unused code from here?